packages/expect/__typechecks__/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@tsconfig/node10/tsconfig.json",
+  "compilerOptions": {
```

Comment:
in addition to the 3 you have, maybe add `"moduleResolution": "node"` and `"esModuleInterop": false`. These might be default, but I'd like to be explicit